webapp/src/components/Wallet/AccountStatus.tsx

```diff
@@ -420,6 +421,15 @@ const AccountStatus: React.FC<AccountStatusProps> = ({
     return <WalletCopyIcon className="far fa-clone" state={copyState} />;
   };
 
+  const renderAirdropButton = () => {
```

Comment:
What does this have to do with the network switcher..?

Reply:
This was part of the mocks because we removed the airdrop button from the header